uvicorn/main.py

```diff
@@ -177,6 +177,12 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     default=True,
     help="Enable/Disable access log.",
 )
+@click.option(
+    "--access-log-format",
+    type=str,
+    default=None,
+    help="Access log format (supports same fields as gunicorn).",
```

Comment:
I don't think we should reference `gunicorn` on the CLI. We can have: "Log format for the access logger, check documentation for details", or something shorter?

Reply:
Fixed!